File: algorithm/python/771.py
class Solution:
    def numJewelsInStones(self, J, S):
        """
        :type J: str
        :type S: str
        :rtype: int
        """
        rtn = 0
        for stone in S:
            for j in J:
                if stone == j:
                    rtn = rtn + 1
        return rtn

File: algorithm/golang/771.go
func numJewelsInStones(J string, S string) int {
    rtn := 0
    for i:=0; i<len(S); i++ {
        for j:=0; j<len(J); j++ {
            if S[i] == J[j] {
                rtn++
            }
        }
    }
    return rtn;
}